httpserver/setup_teardown_test.go

```diff
@@ -56,6 +56,7 @@ func testSetup(t *testing.T) {
 		"Cluster.HeartBeatMissLimit=3",
 		"Cluster.MessageQueueDepthPerPeer=4",
 		"Cluster.MaxRequestDuration=1s",
+		"Cluster.LivenessCheckerEnable=true",
```

Comment:
I'm wondering if we might want to always disable LivenessChecker in our test setup's...
I do not think any of the "unit tests" actually test this... so its fine either way...

Reply:
I left it enabled to, hopefully, reduce "bit rot" so it gets at least little bit of exercise.

Though i'm not sure how useful that is in a "single node" cluster where it has nobody to talk to and none of the tests try to talk to it.  But at least the init/deinit code gets called.

Since you say it's fine either way, i'll leave this enabled.

Reply:
i thought i had already mentioned why i disabled the liveness detector ... for some damn reason it causes a panic when the number of volumes served by a proxyfs process goes beyond ~1,500 volumes.  it looks like this:

```
panic: sendMsg() called for excessive len(msgBuf) == 361741 (must be <= 352410)
--
goroutine 1907977 [running]:
github.com/swiftstack/ProxyFS/liveness.doFollower()
/home/swift/code/ProxyFS/src/github.com/swiftstack/ProxyFS/liveness/states.go:373 +0x1202
github.com/swiftstack/ProxyFS/liveness.stateMachine()
/home/swift/code/ProxyFS/src/github.com/swiftstack/ProxyFS/liveness/states.go:16 +0x27
created by github.com/swiftstack/ProxyFS/liveness.(*globalsStruct).SignaledFinish
/home/swift/code/ProxyFS/src/github.com/swiftstack/ProxyFS/liveness/config.go:840 +0x12f5
", "stdout": ""
```

i did a little bit of digging before deciding to punt the issue (by disabling liveness checker) and move on.

i ran wireshark to collect all the packets sent on that port.  i was anticipating that they would actually grow in size until they eventually hit the limit of 352,410 and i wanted to see what was filling them up.  instead, they stubbornly stayed small as i added volumes(1,400 bytes?) until proxyfs paniced with the same error.